libraries/AP_Notify/Display.h

```diff
@@ -36,6 +39,14 @@ class Display: public NotifyDevice {
     uint8_t _movedelay; // ticker delay before shifting after new message displayed
     uint8_t _screenpage;
 
+    void update_scr_screen(void);
+
+    // stop scripting override if we have not recieved anything for 5 sec
+    static const uint16_t _script_timeout_ms = 5000;
```

Comment:
this could be a #define, or just hard code it with a comment if its only used in one spot

Reply:
Static const is generally better then `#define`